### PR TITLE
Add SSH and SymLink how-tos

### LINKED ISSUE #30: Add link to setting up SSH
@steeleb

### PR DESCRIPTION
**Pull request by steeleb**
Hey Kristen! 

I just added some text to the Best Practices doc for #30.

I rendered the doc on my end and things look good for the edits I added. Please review for grammar/spelling/content. Feel free to suggest changes!

Thank you!

Closes #30

### CODE REVIEW

File: 02-Workflow-Overview.Rmd [hunk 1]
```diff
@@ -100,6 +113,24 @@ your
...
+While you can use either HTTPS or SSH authentication to connect your machine to
+your GitHub repositories, we now suggest using SSH authentication, which is a
+bit more secure.
+
+In order to use the SSH cloning method above, you will need to set up a SSH
+token and agent on your machine. GitHub has a great overview of SSH protocol
+[here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh).
+
+To quickly get started with SSH follow the directions in the sections
+'[Checking for existing SSH
```
Comment (KECognac) [inline 02-Workflow-Overview.Rmd]:
```suggestion
"[Checking for existing SSH
```

File: 02-Workflow-Overview.Rmd [hunk 2]
```diff
@@ -100,6 +113,24 @@ your
...
+keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys)',
```
Comment (KECognac) [inline 02-Workflow-Overview.Rmd]:
```suggestion
keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys)",
```

**Review by KECognac — APPROVED** [review]
Hey B, I changed single quotes to double on lines 127 and 128. Can you update the others you coding genius?

You're free to merge this after making changes.

**Comment by steeleb** [comment]
Thank you!

Merging!

**Pull request merged**